onnxruntime/core/framework/node_unit.cc

```diff
@@ -272,6 +273,20 @@ NodeUnit::NodeUnit(const GraphViewer& graph_viewer, const QDQ::NodeGroup& node_g
   }
 }
 
+NodeUnit::NodeUnit(gsl::span<const Node* const> dq_nodes, const Node& target_node,
+                   gsl::span<const Node* const> q_nodes, Type type,
```

Comment:
Rename type to unit_type if it is UnitType.

Reply:
The type is `Type`, but I renamed the variable to `unit_type` since it matches the `UnitType()` method name.